core/server/worker/src/main/java/alluxio/worker/netty/NettyDataServer.java

```diff
@@ -47,6 +47,10 @@
   private final ServerBootstrap mBootstrap;
   private final ChannelFuture mChannelFuture;
   private final SocketAddress mSocketAddress;
+  private final long mQuietPeriodMs =
```

Comment:
this is not a member variable so that the values configured by the local test cluster are reset if an integration test ends up calling `ConfigurationTestUtils.resetConfiguration`